ThunderCloud/DateFormatter+Extensions.swift

```diff
@@ -11,13 +11,35 @@ import Foundation
 extension DateFormatter {
     
     /// Standard `iso8601` `DateFormatter`
-    /// Note milliseconds including, for customization over `dateFormat`, see `iso8601(dateFormat:)`
-    static let iso8601: DateFormatter = {
+    /// Note milliseconds included, for customization over `dateFormat`, see `iso8601(dateFormat:timeZone:)`
+    static var iso8601: DateFormatter {
+        return iso8601Formatter(
+            dateFormat: "yyyy-MM-dd'T'HH:mm:ss.SSSZZZZZ"
+        )
+    }
+    
+    /// `DateFormatter` has:
+    /// - `.iso8601` `Calendar`
+    /// - "en_US_POSIX" `Locale`
+    /// - Given `timeZone`, defaults to GMT
+    /// - Given `dateFormat`
+    static func iso8601Formatter(dateFormat: String, timeZone: TimeZone? = TimeZone(secondsFromGMT: 0)) -> DateFormatter {
```

Comment:
@simonmitchell The only place of reference sets `TimeZone.current` but we should either use `ThunderBasics` or ideally change to `TimeZone.current` as that's prob preferable to GMT. Though both will work.

Reply:
Going to leave it internal for now, but have made the change to use `.current` :D